FlowCal/plot.py

```diff
@@ -107,9 +108,13 @@ def hist1d(data_list,
         attempts to extract bins from ``data_list[i].domain``.
     histtype : {'bar', 'barstacked', 'step', 'stepfilled'}, str, optional
         Histogram type. Directly passed to ``plt.hist``.
-    normed : bool, optional
+    normed_area : bool, optional
         Flag indicating whether to normalize the histogram such that the
         area under the curve is equal to one.
+    normed_height : bool, optional
+        Flag indicating whether to normalize the histogram such that the
+        sum of all bins' heights is equal to one. Ignored if `normed_area`
+        is True.
```

Comment:
Might be nice to explicitly tie these two normalization techniques to their appropriate PDF or PMF representation, but that's pretty minor.